tests/test_main.py

```diff
@@ -602,7 +602,8 @@ def test_main_noxfile_options_with_sessions_override(
 
 @pytest.mark.parametrize(("isatty_value", "expected"), [(True, True), (False, False)])
 def test_main_color_from_isatty(monkeypatch, isatty_value, expected):
-    monkeypatch.delenv("FORCE_COLOR")
+    if os.getenv("FORCE_COLOR"):
+        monkeypatch.delenv("FORCE_COLOR")
```

Comment:
This can be abbreviated:

```suggestion
    monkeypatch.delenv("FORCE_COLOR", raising=False)
```

Reply:
Cool! I'll do that now